nemo_skills/inference/prompt/utils.py

```diff
@@ -42,7 +42,9 @@ class PromptConfig:
     prefix: str = MISSING
     template: str = MISSING
     context_type: str = "empty"
+    context: Optional[str] = None
```

Comment:
This is being refactored in https://github.com/Kipok/NeMo-Skills/pull/10, so maybe we should switch to that version before merging this. @i-vainn please sync with Daria to make sure the new format supports her use-cases and let's prioritize merging your PR to unblock this one

Reply:
The PR is merged, please integrate it into your branch